src/Http/Http.Extensions/ref/Microsoft.AspNetCore.Http.Extensions.netcoreapp.cs

```diff
@@ -17,6 +17,24 @@ public static partial class HttpRequestExtensions
     {
         public static bool HasJsonContentType(this Microsoft.AspNetCore.Http.HttpRequest request) { throw null; }
```

Comment:
If we move these types then `HttpRequestExtensions` can be removed and `HasJsonContentType` be moved to `HttpRequestJsonExtensions`.